cookbooks/elasticsearch/recipes/default.rb

```diff
@@ -56,3 +45,9 @@
   group "root"
   notifies :run, "execute[reload-sysctl]", :immediately
 end
+
+
+# Pull the Docker image
+execute "pull-elasticsearch-docker-image" do
```

Comment:
I did. In this example it's not necessary to use it, because working directory s unimportant, and it's only one command being run. :+1: